### PR TITLE
Release 1.5.4

### PR DESCRIPTION
**Pull request by tillrohrmann**
Release announcement for Flink 1.5.4

### CODE REVIEW

**Comment by tzulitai** [comment]
+1, LGTM

File: _posts/2018-09-19-release-1.5.4.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10293'>FLINK-10293</a>] -         RemoteStreamEnvironment does not forward port to RestClusterClient
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10314'>FLINK-10314</a>] -         Blocking calls in Execution Graph creation bring down cluster
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10328'>FLINK-10328</a>] -         Stopping the ZooKeeperSubmittedJobGraphStore should release all currently held locks
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10329'>FLINK-10329</a>] -         Fail with exception if job cannot be removed by ZooKeeperSubmittedJobGraphStore#removeJobGraph
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-10330'>FLINK-10330</a>] -         Code Compilation failed in local mode
```
Comment (twalthr) [inline _posts/2018-09-19-release-1.5.4.md]: @tillrohrmann This issue has not been fixed yet. I removed the fixed version in JIRA. Please update this entry here.

Comment (tillrohrmann) [reply _posts/2018-09-19-release-1.5.4.md]: Updated the release notes.

**Comment by tillrohrmann** [comment]
Has been merged as part of #124.

**Pull request closed without merging**